configs/staff_hours.yaml

```diff
@@ -0,0 +1,37 @@
+# Authoritative staff hours file.
+#
+# The online staff hours page is dynamically generated based on the
+# contents of this file.
+#
+# Be sure to use spaces instead of tabs for indentation of this file.
+#
+# An example staff hours entry looks like:
+#
+# staff-hours:
+#  Monday:
+#   - time: ['11:00', '13:00']
+#     staff: ["gstaff"]
+#  Tuesday:
+#  ...
+---
+
+staff-hours:
+  Monday:
+  Tuesday:
+  Wednesday:
+  Thursday:
+  Friday:
+  Saturday:
+  Sunday:
+
+staff-positions:
```

Comment:
This is fine for now, but we should plan to replace this with data from officers (see #3)